### PR TITLE
PT-820: Add more validations during import

### PR DESCRIPTION
**Pull request by eMazeika**

### CODE REVIEW

**Review by akak1977 — APPROVED** [review]

File: src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogImporter.cs [hunk 1]
```diff
@@ -140,7 +140,12 @@ public Task DoImportAsync(Stream inputStream, CsvImportInfo importInfo, Action<E
                     }
                     catch (Exception ex)
                     {
-                        var error = ex.Message;
+                        var error = ex.Message;
+                        if (ex.InnerException?.Message != null)
+                        {
+                            error += " " + ex.InnerException.Message;
```
Comment (ksavosteev) [inline src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogImporter.cs:146]: error = $"{error}: {ex.InnerException.Message}";

**Review by ksavosteev — APPROVED** [review]

File: src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogImporter.cs [hunk 2]
```diff
@@ -359,7 +364,7 @@ private async Task ResolvePropertyDictionaryItems(List<CsvProduct> csvProducts,
 
             foreach (var dictProperty in csvProducts.SelectMany(x => x.Properties).Where(x => x.Dictionary && x.Values?.Any(v => v != null) == true))
             {
-                foreach (var propertyValue in dictProperty.Values.Where(x => x.Value != null))
+                foreach (var propertyValue in dictProperty.Values.Where(x => !string.IsNullOrWhiteSpace(x.Value?.ToString())))
```
Comment (ksavosteev) [inline src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogImporter.cs:367]: Do we really need ToString here?

Comment (eMazeika) [reply src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogImporter.cs:367]: method accepts string only, not object

File: src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogImporter.cs (hunk 1 above)
Comment (eMazeika) [reply src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogImporter.cs]: sure

**Pull request merged**